M2/Macaulay2/packages/Macaulay2Doc/functions/lift-doc.m2

```diff
@@ -122,5 +122,5 @@ document {
      .0001^QQ
      .0001_QQ
      ///,
-     SeeAlso => {baseRings,liftable,promote}
+     SeeAlso => {baseRings,liftable,promote,setupLift}
```

Comment:
I would even put `setupLift` under `Subnodes` of `lift`, just to give it more structure.

Reply:
Same with `liftable`.